packages/suite/src/views/wallet/sign-verify/components/Navigation.tsx

```diff
@@ -1,4 +1,22 @@
-import { WalletLayoutNavigation, WalletLayoutNavLink } from 'src/components/wallet';
+import styled from 'styled-components';
+import { NavigationTab } from './NavigationTab';
+
+const Container = styled.div`
+    display: flex;
+    width: 100%;
+    min-height: 57px;
+    padding: 0 25px;
+    overflow-x: auto;
+    scrollbar-width: none; /* Firefox */
```

Comment:
Maybe, it would be good to somewhere separate the CSS code for removing scrollbar and then just use it here.